react-frontend/src/App.tsx

```diff
@@ -10,7 +10,6 @@ const getUserQueryDocument = graphql(/* GraphQL */`
   query GetUser($userId: ID!) {
     user(id: $userId) {
       id
-      username
```

Comment:
I'd like this sample query to hit both backend

Reply:
same